ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/SourceDescription.java

```diff
@@ -104,8 +118,61 @@ public SourceDescription(
         Objects.requireNonNull(queryOffsetSummaries, "queryOffsetSummaries"));
     this.sourceConstraints =
         ImmutableList.copyOf(Objects.requireNonNull(sourceConstraints, "sourceConstraints"));
+    this.clusterErrorStats = clusterErrors;
+    this.clusterStatistics = clusterStats;
   }
 
+  // CHECKSTYLE_RULES.OFF: ParameterNumberCheck
+  @SuppressWarnings("checkstyle:Indentation")
```

Comment:
Do we want to suppress that? Or can we switch the indentation so it doesn't complain?

Reply:
Oh this was accidental!